pkg/term/env.go

```diff
@@ -71,11 +75,21 @@ func FromEnv() Term {
 	}
 }
 
+// In is the reader reading from standard input.
+func (t Term) In() io.Reader {
+	return t.in
+}
+
 // Out is the writer writing to standard output.
 func (t Term) Out() io.Writer {
 	return t.out
 }
 
+// Err is the writer writing to standard error.
+func (t Term) Err() io.Writer {
```

Comment:
Nit: `Err` is sometimes in Go a method that returns an `error` object, for example `Context.Err()`

To be more clear about the purpose of this writer, could we maybe call it `ErrOut`?